pkg/integrations/kafka_lag_exporter/kafka_lag_exporter.go

```diff
@@ -0,0 +1,173 @@
+	KafkaVersion:            sarama.V2_0_0_0.String(),
+	UseZooKeeperLag:         false,
+	Labels:                  "dev-cluster",
+	MetadataRefreshInterval: "1m",
+	AllowConcurrent:         true,
+	MaxOffsets:              1000,
+	PruneIntervalSeconds:    30,
+}
+
+type Config struct {
+	Common config.Common `yaml:",inline"`
+
+	// Exporter configuration
+
+	//Address array (host:port) of Kafka server
+	KafkaUri []string `yaml:"kafka_uris,omitempty"`
+
+	//Connect using SASL/PLAIN
+	UseSASL bool `yaml:"use_sasl,omitempty"`
+
+	//Only set this to false if using a non-Kafka SASL proxy
+	UseSASLHandshake bool `yaml:"use_sasl_handshake,omitempty"`
+
+	//SASL user name
+	SaslUsername string `yaml:"sasl_username,omitempty"`
+
+	//SASL user password
+	SaslPassword string `yaml:"sasl_password,omitempty"`
+
+	//The SASL SCRAM SHA algorithm sha256 or sha512 as mechanism
+	SaslMechanism string `yaml:"sasl_mechanism,omitempty"`
+
+	//Connect using TLS
+	UseTLS bool `yaml:"use_tls,omitempty"`
+
+	//The optional certificate authority file for TLS client authentication
+	TlsCAFile string `yaml:"tls_cafile,omitempty"`
+
+	//The optional certificate file for TLS client authentication
+	TlsCertFile string `yaml:"tls_certfile,omitempty"`
+
+	//Password for the certificate file for TLS client authentication
+	TlsCertFilePassword string `yaml:"tls_certfile_password,omitempty"`
+
+	//The optional key file for TLS client authentication
+	TlsKeyFile string `yaml:"tls_keyfile,omitempty"`
+
+	//Password for the key file for TLS client authentication
+	TlsKeyFilePassword string `yaml:"tls_keyfile_password,omitempty"`
+
+	//If true, the server's certificate will not be checked for validity. This will make your HTTPS connections insecure
+	TlsInsecureSkipTLSVerify bool `yaml:"tls_insecure_skip_tlsverify,omitempty"`
+
+	//Kafka broker version
+	KafkaVersion string `yaml:"kafka_version,omitempty"`
+
+	//if you need to use a group from zookeeper
+	UseZooKeeperLag bool `yaml:"use_zookeeper_lag,omitempty"`
+
+	//Address array (hosts) of zookeeper server.
+	UriZookeeper []string `yaml:"zookeeper_uris,omitempty"`
+
+	//Kafka cluster name
+	Labels string `yaml:"kafka_cluster_name,omitempty"`
+
+	//Metadata refresh interval
+	MetadataRefreshInterval string `yaml:"metadata_refresh_interval,omitempty"`
+
+	//If true, all scrapes will trigger kafka operations otherwise, they will share results. WARN: This should be disabled on large clusters
+	AllowConcurrent bool `yaml:"allow_concurrency,omitempty"`
+
+	//Maximum number of offsets to store in the interpolation table for a partition
+	MaxOffsets int `yaml:"max_offsets,omitempty"`
+
+	//How frequently should the interpolation table be pruned, in seconds
+	PruneIntervalSeconds int `yaml:"prune_interval_seconds,omitempty"`
+
+	//Regex filter for topics to be monitored
+	TopicsFilter string `yaml:"topics_filter_regex,omitempty"`
+
+	//Regex filter for consumer groups to be monitored
+	GroupFilter string `yaml:"groups_filter_regex,omitempty"`
```

Comment:
Should these have defaults in `DefaultConfig` matching the defaults the docs say they have?